src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -302,9 +307,9 @@
     </fields>
     <fields>
         <fullName>RecurringType__c</fullName>
-        <description>Default value is Open, meaning a recurring donation with no end date.</description>
+        <description>Select Open for an ongoing Recurring Donation with no specific number of installments. Select Fixed for a Recurring Donation that has a specific number of installments. For Fixed, you must also populate the Number of Planned Installments field.</description>
```

Comment:
@sam-knox This is the updated description/helptext we discussed on the work item, but wanted to make sure it looks good before moving forward.